app-devel/judy/autobuild/build

```diff
@@ -0,0 +1,5 @@
+abinfo "Building judy ..."
+make -j1
```

Comment:
Use NOPARALLEL=1 and explain why in a FIXME comment.

Reply:
Added.